### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by theduc151**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
Explain clearly and briefly what this PR does. Make a meaningful sentence instead of "this PR solve ticket #xxx"

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
- Screenshots
- link to demo
- instruction on how to test this

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: test/fixtures/categories.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (rikai-hai) [inline test/fixtures/categories.yml]: xóa file dạng này

File: test/models/category_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (rikai-hai) [inline test/models/category_test.rb]: xóa hết file test

File: db/migrate/20220407062726_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.boolean :is_admin
```
Comment (rikai-hai) [inline db/migrate/20220407062726_create_users.rb]: thiếu default: false

File: db/migrate/20220407062906_create_orders.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class CreateOrders < ActiveRecord::Migration[6.0]
+  def change
+    create_table :orders do |t|
+      t.references :user, foreign_key: true
+      t.references :payment_method, foreign_key: true
+      t.text :status
```
Comment (rikai-hai) [inline db/migrate/20220407062906_create_orders.rb]: status sao lại text hè, dùng integer chứ
https://betterprogramming.pub/how-to-use-enums-in-rails-6-87600e292476#:~:text=In%20Ruby%20on%20Rails%2C%20an,for%20enums%20in%20Rails%204.1.

File: db/migrate/20220407062935_create_rates.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+class CreateRates < ActiveRecord::Migration[6.0]
+  def change
+    create_table :rates do |t|
+      t.references :user, foreign_key: true
+      t.string :rateable_id
+      t.string :rateable_type
+      t.float :point
```
Comment (rikai-hai) [inline db/migrate/20220407062935_create_rates.rb]: default: 0

File: db/migrate/20220407062958_create_order_details.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+class CreateOrderDetails < ActiveRecord::Migration[6.0]
+  def change
+    create_table :order_details do |t|
+      t.references :product, foreign_key: true
+      t.references :order, foreign_key: true
+      t.integer :quantity
```
Comment (rikai-hai) [inline db/migrate/20220407062958_create_order_details.rb]: default: 1
price: default 0

File: db/migrate/20220407063022_create_images.rb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+class CreateImages < ActiveRecord::Migration[6.0]
+  def change
+    create_table :images do |t|
+      t.string :imageable_id
+      t.string :imageable_type
+
+      t.timestamps
```
Comment (rikai-hai) [inline db/migrate/20220407063022_create_images.rb]: chỗ này thì ko cần

File: db/migrate/20220407062432_create_categories.rb [hunk 8]
```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration[6.0]
+  def change
+    create_table :categories do |t|
+      t.string :name
```
Comment (rikai-hai) [inline db/migrate/20220407062432_create_categories.rb:4]: bảng giữa của category và product đâu hè

File: db/migrate/20220407062906_create_orders.rb (hunk 4 above)
Comment (theduc151) [reply db/migrate/20220407062906_create_orders.rb]: status có cần default không vậy ???

**Pull request merged**